src/Api/Extension/FilterUserCollectionsExtension.php

```diff
@@ -53,6 +53,8 @@ private function filterPersonalData(QueryBuilder $queryBuilder, string $rootAlia
     {
         $beneficiary = $user->getSubjectBeneficiaire();
         if (!$beneficiary) {
+            $queryBuilder->andWhere('1 = 0');
+
             return;
```

Comment:
It is a little bit weird.
If we do this, maybe we should have a method with an explicit name